db/migrate/20180416213522_add_notification_schedule_to_users.rb

```diff
@@ -0,0 +1,7 @@
+class AddNotificationScheduleToUsers < ActiveRecord::Migration
+  def change
+    change_table :users do |t|
+      t.boolean :notify_daily, null: false, default: true
```

Comment:
I'd rather go with descriptive strings, since there isn't a particular efficiency or space constraint and it's unlikely someone will want their email delivered every 37 minutes, or whatever.